src/MICore/LaunchOptions.xsd.types.designer.desktop.cs

```diff
@@ -0,0 +1,1161 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// 
+// This source code was auto-generated by xsd, Version=4.6.81.0.
```

Comment:
Do we need to have multiple copies of this file? Its going to be annoying to maintain more than one.